### DISCUSSION TITLE
Can we use pnpm 9 with node 22?

### DISCUSSION THREAD

**Discussion by rnike** [pnpm/pnpm#8909] [Q&A] [answered]
Hello, we are currently with pnpm version 9.15.1, wanted to know if we can upgrade our node.js to [22.12.0 (LTS)](https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V22.md)

We are asking because we cannot find Node.js 22 on the [compatibility table](https://pnpm.io/installation#compatibility)
![Screenshot 2024-12-26 at 2 13 42 PM](https://github.com/user-attachments/assets/8617b161-3cca-43a6-aff3-f0ba21f9986c)

**Comment by zkochan** [comment, marked as answer]
yes, you can. We run the tests on node.js 22

**Reply by rnike** [reply]
> yes, you can. We run the tests on node.js 22

thanks a lot
